### PR TITLE
feat(kit): `Push` add new component

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/demo/src/modules/components/push/push.template.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            [content]="example3"
+        >
+            <tui-push-example-3></tui-push-example-3>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab="Подключение">
+        <ol class="b-demo-steps">
+            <li>
+                Для использования импортировать модуль
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/push/push.template.html:36]: eng

**Review by splincode — APPROVED** [review]

File: projects/kit/components/push/push-alert.component.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import {TuiPushAlertDirective} from './push-alert.directive';
+
+@Component({
+    selector: `tui-push-alert`,
+    templateUrl: `./push-alert.template.html`,
+    styleUrls: [`./push-alert.style.less`],
+    animations: [tuiFadeIn, tuiSlideInRight, tuiHeightCollapse],
+    host: {role: `alert`},
+})
+export class TuiPushAlertComponent {
```
Comment (nsbarsukov) [inline projects/kit/components/push/push-alert.component.ts:22]: `changeDetection: ChangeDetectionStrategy.OnPush` ?

File: projects/kit/components/push/push.template.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        *ngIf="timestamp"
+        class="t-time"
+        [textContent]="timestamp | tuiFormatDate | async"
+    ></span>
+</div>
+<h3 class="t-heading">{{ heading }}</h3>
+<div class="t-content">
+    <ng-content></ng-content>
+</div>
+<div class="t-actons">
```
Comment (nsbarsukov) [inline projects/kit/components/push/push.template.html:26]:
```suggestion
<div class="t-actions">
```
?

File: projects/kit/components/push/push.style.less [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    display: -webkit-box;
+
+    /* autoprefixer: ignore next */
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    word-break: break-word;
+    overflow: hidden;
+}
+
+.t-actons {
```
Comment (nsbarsukov) [inline projects/kit/components/push/push.style.less:73]:
```suggestion
.t-actions {
```
?

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/app/pages.ts [hunk 5]
```diff
@@ -596,6 +596,12 @@ export const pages: TuiDocPages = [
         keywords: `уведомление, нотификация, бабл, облачко, alert, notification`,
         route: `/components/notification`,
     },
+    {
+        section: $localize`Patterns`,
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/demo/src/modules/app/pages.ts:600]: Nope, my mistake from initially working in proprietary. Fixed.

File: projects/demo/src/modules/app/pages.ts [hunk 6]
```diff
@@ -596,6 +596,12 @@ export const pages: TuiDocPages = [
...
+        title: `Push`,
+        keywords: `push, пуш, нотификаци, notification, alert`,
+        route: `/push`,
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/demo/src/modules/app/pages.ts:603]: Fixed

File: projects/demo/src/modules/components/push/push.template.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-push
+                [heading]="heading"
+                [type]="type"
+                [timestamp]="timestamp"
+                (close)="onClose()"
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/demo/src/modules/components/push/push.template.html:39]: Fixed

File: projects/kit/components/push/push-alert.template.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
+<tui-push
+    *ngIf="!isDirective; else content"
+    [heading]="context.heading"
+    [type]="context.type"
+    [timestamp]="context.timestamp"
+    (close)="context.$implicit.complete()"
+>
+    <img
+        *ngIf="context.image"
+        alt=""
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/kit/components/push/push-alert.template.html:10]: This image serves decorative purposes. Let's not bloat inputs for that unless we have to.

File: projects/kit/components/push/push-alert.component.ts (hunk 2 above)
Comment (waterplea) [reply projects/kit/components/push/push-alert.component.ts:22]: Fixed

File: projects/kit/components/push/push.template.html (hunk 3 above)
Comment (waterplea) [reply projects/kit/components/push/push.template.html:26]: Fixed

File: projects/kit/components/push/push.style.less (hunk 4 above)
Comment (waterplea) [reply projects/kit/components/push/push.style.less:73]: Fixed

File: projects/demo/src/modules/components/push/examples/1/index.less [hunk 9]
```diff
@@ -0,0 +1,20 @@
+@import 'taiga-ui-local';
+
+:host {
+    display: block;
+}
+
+.wrapper {
+    .shadow();
+    width: 22.5rem;
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/demo/src/modules/components/push/examples/1/index.less:9]: Done

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**